src/Campaigns/Actions/ConvertQueryDataToCampaign.php

```diff
@@ -33,7 +33,7 @@ public function __invoke(object $queryObject): Campaign
             'goal' => (int)$queryObject->goal,
             'goalType' => new CampaignGoalType($queryObject->goalType),
             'startDate' => Temporal::toDateTime($queryObject->startDate),
-            'endDate' => Temporal::toDateTime($queryObject->endDate),
+            'endDate' => $queryObject->endDate ? Temporal::toDateTime($queryObject->endDate) : $queryObject->endDate,
```

Comment:
To be safe, I would probably just default to null explicitly 

```suggestion
            'endDate' => $queryObject->endDate ? Temporal::toDateTime($queryObject->endDate) : null,
```

Reply:
Resolved in 43c8a514386deb2e26b564708d89e2bfc1464b82